mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
```

Comment:
The code snippet provided is a part of the `deleteOrder` function. 

Here is the review:

The code snippet is correct and follows best practices. It returns a clear and informative JSON response with a status code of 404 when the order does not exist. 

However, as mentioned in the retrieved context, it would be better to include a return statement after sending the response for a non-existent order to prevent the code from continuing to execute and sending another response with "Order deleted successfully". 

So, the corrected code snippet should be:

`if (!order) { return res.status(404).json({ message: "Order does not exist" }); }`